src/SPC/builder/extension/openssl.php

```diff
@@ -26,6 +26,27 @@ public function patchBeforeMake(): bool
     public function getUnixConfigureArg(bool $shared = false): string
     {
         $openssl_dir = $this->builder->getPHPVersionID() >= 80400 ? '' : ' --with-openssl-dir=' . BUILD_ROOT_PATH;
-        return '--with-openssl=' . ($shared ? 'shared,' : '') . BUILD_ROOT_PATH . $openssl_dir;
+        $args = '--with-openssl=' . ($shared ? 'shared,' : '') . BUILD_ROOT_PATH . $openssl_dir;
+        if (
+            $this->builder->getPHPVersionID() >= 80500 &&
+            ($ver = $this->builder->getLib('openssl')->getLibVersion()) &&
+            version_compare($ver, '3.2.0', '>=')
```

Comment:
we can skip this check given our download will not just decide to revert to an old version

there are many places where we (or php) requires a certain minimum version